src/components/molecules/Tag/Tag.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import TagIcon from '../../atoms/TagIcon';
+import theme from '../../../style/theme';
+
+const Wrapper = styled.div`
+  display: inline-flex;
+  border-radius: 2vh;
+  padding: 0.9vh 1.8vh;
+  background-color: ${({ isAddingTag, tagColor }) =>
+    isAddingTag ? `white` : tagColor};
+  border: ${({ isAddingTag, tagColor }) =>
+    isAddingTag ? `solid 0.2vh ${tagColor}` : ''};
```

Comment:
```suggestion
  ${({ isAddingTag, tagColor }) =>
    isAddingTag && `border: solid 0.2vh ${tagColor}`};
```

삼항연산자를 굳이 쓸 필요가 없을 것 같습니다!

Reply:
아하 저 항상 삼항 대체?로 && 사용하는게 헷갈렸는데 짚어주셔서 감사드려요!!👍👍